src/routes/safe/components/Settings/Advanced/RemoveModuleModal.tsx

```diff
@@ -36,6 +36,7 @@ export const RemoveModuleModal = ({ onClose, selectedModulePair }: RemoveModuleM
   const classes = useStyles()
 
   const safeAddress = useSelector(safeAddressFromUrl)
+  const safeVersion = useSelector(currentSafeCurrentVersion) as string
```

Comment:
here, I think the `safeAddressFromUrl` shouldn't be used and given that you need two attributes from the current safe. Maybe would be better to use `currentSafe` selector and destructure from there, wdyt?

something similar to https://github.com/gnosis/safe-react/blob/e9806b0918f4e7fc34603b5f9b24f0bc21858a80/src/routes/safe/components/Settings/SpendingLimit/NewLimitModal/Review.tsx#L156

Reply:
Sometimes is difficult to find the balance on when to use the specific prop or to use currentSafe for everything ;)
Will change them